test/sqllogictest/mz-global-id-funcs.slt

```diff
@@ -142,3 +142,8 @@ query T
 SELECT mz_internal.mz_name_to_global_id(null, ARRAY['pg_catalog'], 'int4')
 ----
 s20
+
+# Tables have no subsources
+query TTT
+SELECT * FROM mz_internal.mz_get_subsources('t');
+----
```

Comment:
Can you add a test where the table has some views or indexes to make sure those are mistakenly being show.